src/test/resources/Json.request/All/WithdrawAdviceAPI/Scenario_WithdrawAdvice_EmptyString_204.json

```diff
@@ -0,0 +1,3 @@
+{
+  "withdrawReason": ""
```

Comment:
we would get bad request for empty string as we have length limit, may be pass as space " ".